apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -43,188 +36,197 @@ export interface RepeatingCustomEvent {
 interface CustomEventDialogProps {
     customEvent?: RepeatingCustomEvent;
     onDialogClose?: () => void;
-    scheduleNames: string[];
```

Comment:
Not sure if this was intentional, but this is causing an error in CalendarToolbar.tsx